.travis.yml

```diff
@@ -1,16 +1,28 @@
 language: python
 python:
-- '3.6'
+  - '3.6'
+node_js:
+  - node
+addons:
+  apt:
+    sources:
+      - sourceline: deb https://dl.yarnpkg.com/debian/ stable main
+      key_url: https://dl.yarnpkg.com/debian/pubkey.gpg
+    packages:
+      - yarn
 sudo: true
 dist: trusty
+group: deprecated-2017Q3
 install:
-- sudo sed -i 's/trusty/yakkety/' /etc/apt/sources.list
-- sudo rm -rf /etc/apt/sources.list.d/*
-- sudo apt-get update -qq
-- sudo apt-get install -qq pandoc ca-certificates
-- pip install -r requirements.txt
+  - yarn
+  - sudo sed -i 's/trusty/yakkety/' /etc/apt/sources.list
+  - sudo rm -rf /etc/apt/sources.list.d/*
+  - sudo apt-get update -qq
+  - sudo apt-get install -qq pandoc ca-certificates
+  - pip install -r requirements.txt
 script:
-- ":"
+  - yarn lint
+  - ":"
```

Comment:
this is a no-op, we don't need to keep it here, it was there just to unset the default value.